tests/unit/test_bruteforce.cpp

```diff
@@ -1315,7 +1545,7 @@ TEST_F(BruteForceTest, testCosine) {
 
 TEST_F(BruteForceTest, testSizeEstimation) {
     size_t dim = 128;
-    size_t n = 10000;
+    size_t n = 0;
```

Comment:
why this change?

Reply:
To check if the estimation wont include the allocation overhead of id2labelMapping.
size estimation with an initial capacity is covered in the next test